### PR TITLE
Upgrade Ancient Religions from alpha to v0.36.1 #23

### PR DESCRIPTION
**Pull request by rquinio**
Integration of Ancient Religions Reborn v0.36.1, which is much more polished and stable than the older alpha version !
Changelog for ACR is accessible [here](https://github.com/xirsoi/ancientreligions/blob/master/ACR_Changelog.txt).

Please review the changes, especially on 00_religions.txt and other common files.
Note that lots of ACR mod files have been refactored (either merged, renamed with ACR_ prefix or moved to ACR sub-folders) which simplify the integration.

Integration impacts:
- Rename kemetic_pagan to egyptian_pagan to ease integration with ACR
- Update religion icon strip to include hellenic heresy and celtic pagan icons from ACR #56
- Simplify holy war to all reformed pagans (plus aztec and mithraic_pagan)
- Simplify convert_to_reformed decision using abandon_heresy command
- Workaround nickname conflicts with Siren mod
- Remove ACR localizations from inegrated_mod_local.csv
- Remove duplicate religion names and descriptions with ACR from WtWSMS_religions.csv
- Remove event_pictures moved to ACR sub-folder

I've done some quick checks as Celtic, Druidic and Kemetic characters and didn't spot integration issues.

### CODE REVIEW

**Comment by rquinio** [comment]
Btw, on ACR has_coa_on_barony_only = no was activated at pagan_group level, to make use of the nice frames and CoA patterns for Celtic/Hellenic/Kemetic, and to avoid using Christian flags while playing as a pagan, which break immersion.

As I wasn't sure how much of the pagan flags (Germanic, Slavic, African,...) are historical for the timeframe vs placeholder, I didn't activate it here.

File: WTWSMS/localisation/WtWSMS_religions.csv [hunk 1]
```diff
@@ -476,7 +449,7 @@ duke_title_ruler_consort_female_hellenic_pagan;Mistress;Mistress;Mistress;;Mistr
 king_title_ruler_consort_female_hellenic_pagan;Mistress;Mistress;Mistress;;Mistress;;;;;;;x
 emperor_title_ruler_consort_female_hellenic_pagan;Mistress;Mistress;Mistress;;Mistress;;;;;;;x
 temple_baron_hellenic_pagan;Priest;;;;;;;;;;;x
-temple_baron_hellenic_pagan_female;Vestal Virgin;;;;;;;;;;;x
+#temple_baron_hellenic_pagan_female;Vestal Virgin;;;;;;;;;;;x
```
Comment (rquinio) [inline WTWSMS/localisation/WtWSMS_religions.csv]: Default to priestess, as Vestals have specific events/minor titles/trait in ACR

File: WTWSMS/localisation/WtWSMS_religions.csv [hunk 2]
```diff
@@ -421,15 +401,8 @@ job_chancellor_celtic_pagan;Seannsalair;;;;;;;;;;;x
 job_spymaster_celtic_pagan;Brath;;;;;;;;;;;x
 job_treasurer_celtic_pagan;Rianadair;;;;;;;;;;;x
 job_marshal_celtic_pagan;Curaidh;;;;;;;;;;;x
-job_spiritual_celtic_pagan;Court Druid;;;;;;;;;;;x
+#job_spiritual_celtic_pagan;Court Druid;;;;;;;;;;;x
```
Comment (rquinio) [inline WTWSMS/localisation/WtWSMS_religions.csv]: Use ACR localization to avoid confusion, as Court Druid is a minor title required for some decisions.

**Comment by loup99** [comment]
I somehow missed this due to some settings (sorry), but I will for sure take a look at it!

**Comment by loup99** [comment]
The "gaulish" culture should not be needed in this mod, we have "Gallic". Or what is the exact purpose of it?

**Comment by loup99** [comment]
Ok, so i looked througfh it al, and it looks good, except for the Gaulish culture. The event ancrel.9010 that is fired on startup, is it really necessary for this mod? 

EDIT: How do I make changes to this "pull request"?

**Comment by rquinio** [comment]
Gaulish culture is a revivalist culture for Continental Celtic pagans. But the revival events are not in the mod yet, so it could be ignored in this version ! I'm not sure how different it is from Gallic, I'll ask ^^

You can make changes by pushing directly to the branch [23-acr-update]( https://github.com/rickinator9/WTWSMS/tree/23-acr-update) ;)

**Comment by loup99** [comment]
Ok, I will look into changing that, and then merge.

**Comment by loup99** [comment]
OK, I will merge then.

**Pull request merged**